src/Rector/v11/v5/SubstituteBackendTemplateViewWithModuleTemplateRector.php

```diff
@@ -219,7 +219,9 @@ private function callSetContentAndGetContent(ClassMethod $classMethod): void
     {
         $classMethodName = (string) $this->getName($classMethod->name);
 
-        if (! str_ends_with($classMethodName, 'Action')) {
+        if (! str_ends_with($classMethodName, 'Action')
+            || 'initializeAction' === $classMethodName
+            || 'initializeIndexAction' === $classMethodName) {
```

Comment:
no, this won't fix it. The pattern is "initialize" + whatever + "Action". So we just need to test if the action starts with "initialize" and ends with "Action".